app/Commands/Dev/Latex/Export.hs

```diff
@@ -0,0 +1,59 @@
+module Commands.Dev.Latex.Export
+  ( module Commands.Dev.Latex.Export,
+    module Commands.Dev.Latex.Export.Options,
+  )
+where
+
+import Commands.Base
+import Commands.Dev.Latex.Export.Options
+import Data.Text qualified as Text
+import Juvix.Compiler.Backend.Latex.Translation.FromScoped.Source
+import Juvix.Compiler.Concrete.Language
+import Juvix.Compiler.Concrete.Translation.FromParsed.Analysis.Scoping qualified as Scoper
+
+runCommand :: (Members AppEffects r) => ExportOptions -> Sem r ()
+runCommand ExportOptions {..} = do
+  res :: Scoper.ScoperResult <- silenceProgressLog (runPipelineNoOptions (Just _exportInputFile) upToScopingEntry)
+  let m :: Module 'Scoped 'ModuleTop = res ^. Scoper.resultModule
+      c :: Maybe Comments = guard (not _exportNoComments) $> Scoper.getScoperResultComments res
+      ltx :: Text =
+        Text.unlines
+          . sublist (pred <$> _exportFromLine) (pred <$> _exportToLine)
+          . Text.lines
+          $ moduleToLatex c m
+  renderStdOut $
+    case _exportMode of
+      ExportStandalone -> standalone ltx
+      ExportRaw -> ltx
+      ExportWrap -> verb ltx
+
+verb :: Text -> Text
+verb code =
+  Text.unlines
```

Comment:
Minor: It might be clearer to use the string interpolation quasi-quote for these text blocks:

https://github.com/anoma/juvix/blob/bd3b7f140104668e72134bf9e9adb138d82baf09/app/Commands/Repl.hs#L53

https://www.stackage.org/haddock/nightly-2024-08-02/string-interpolate-0.3.4.0/Data-String-Interpolate.html

Reply:
done